src/app.js

```diff
@@ -2,6 +2,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+import crypto from 'crypto'
```

Comment:
suggestion: is `crypto` a native node package?  If so, I've found the following convention helpful in identifying when a package originates from node:

```suggestion
import crypto from 'node:crypto'
```